[2020-04-22 17:31:03,631] ERROR Failed to access checkpoint file cleaner-offset-checkpoint in dir /data/sdi1500059/logs/kafka-logs-0 (kafka.log.LogCleaner)
org.apache.kafka.common.errors.KafkaStorageException: Error while reading checkpoint file /data/sdi1500059/logs/kafka-logs-0/cleaner-offset-checkpoint
Caused by: java.nio.file.NoSuchFileException: /data/sdi1500059/logs/kafka-logs-0/cleaner-offset-checkpoint
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
	at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:214)
	at java.nio.file.Files.newByteChannel(Files.java:361)
	at java.nio.file.Files.newByteChannel(Files.java:407)
	at java.nio.file.spi.FileSystemProvider.newInputStream(FileSystemProvider.java:384)
	at java.nio.file.Files.newInputStream(Files.java:152)
	at java.nio.file.Files.newBufferedReader(Files.java:2784)
	at java.nio.file.Files.newBufferedReader(Files.java:2816)
	at kafka.server.checkpoints.CheckpointFile.liftedTree2$1(CheckpointFile.scala:87)
	at kafka.server.checkpoints.CheckpointFile.read(CheckpointFile.scala:86)
	at kafka.server.checkpoints.OffsetCheckpointFile.read(OffsetCheckpointFile.scala:61)
	at kafka.log.LogCleanerManager.$anonfun$allCleanerCheckpoints$2(LogCleanerManager.scala:135)
	at scala.collection.TraversableLike.$anonfun$flatMap$1(TraversableLike.scala:244)
	at scala.collection.Iterator.foreach(Iterator.scala:941)
	at scala.collection.Iterator.foreach$(Iterator.scala:941)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1429)
	at scala.collection.MapLike$DefaultValuesIterable.foreach(MapLike.scala:213)
	at scala.collection.TraversableLike.flatMap(TraversableLike.scala:244)
	at scala.collection.TraversableLike.flatMap$(TraversableLike.scala:241)
	at scala.collection.AbstractTraversable.flatMap(Traversable.scala:108)
	at kafka.log.LogCleanerManager.$anonfun$allCleanerCheckpoints$1(LogCleanerManager.scala:133)
	at kafka.utils.CoreUtils$.inLock(CoreUtils.scala:253)
	at kafka.log.LogCleanerManager.allCleanerCheckpoints(LogCleanerManager.scala:141)
	at kafka.log.LogCleanerManager.$anonfun$grabFilthiestCompactedLog$1(LogCleanerManager.scala:172)
	at kafka.utils.CoreUtils$.inLock(CoreUtils.scala:253)
	at kafka.log.LogCleanerManager.grabFilthiestCompactedLog(LogCleanerManager.scala:169)
	at kafka.log.LogCleaner$CleanerThread.cleanFilthiestLog(LogCleaner.scala:321)
	at kafka.log.LogCleaner$CleanerThread.doWork(LogCleaner.scala:307)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2020-04-22 17:31:33,261] INFO Starting the log cleaner (kafka.log.LogCleaner)
[2020-04-22 17:31:33,322] INFO [kafka-log-cleaner-thread-0]: Starting (kafka.log.LogCleaner)
[2020-04-22 17:31:34,180] INFO Starting the log cleaner (kafka.log.LogCleaner)
[2020-04-22 17:31:34,241] INFO [kafka-log-cleaner-thread-0]: Starting (kafka.log.LogCleaner)
[2020-04-22 17:31:35,204] INFO Starting the log cleaner (kafka.log.LogCleaner)
[2020-04-22 17:31:35,259] INFO [kafka-log-cleaner-thread-0]: Starting (kafka.log.LogCleaner)
[2020-04-22 17:31:35,965] INFO Starting the log cleaner (kafka.log.LogCleaner)
[2020-04-22 17:31:36,024] INFO [kafka-log-cleaner-thread-0]: Starting (kafka.log.LogCleaner)
[2020-04-22 17:31:36,839] INFO Starting the log cleaner (kafka.log.LogCleaner)
[2020-04-22 17:31:36,891] INFO [kafka-log-cleaner-thread-0]: Starting (kafka.log.LogCleaner)
[2020-04-22 17:57:13,259] INFO The cleaning for partition test-part-1-rep-5-0 is aborted and paused (kafka.log.LogCleaner)
[2020-04-22 17:57:13,260] INFO The cleaning for partition test-part-1-rep-5-0 is aborted (kafka.log.LogCleaner)
[2020-04-22 17:57:13,267] INFO The cleaning for partition test-part-1-rep-5-0 is aborted and paused (kafka.log.LogCleaner)
[2020-04-22 17:57:13,268] INFO The cleaning for partition test-part-1-rep-5-0 is aborted (kafka.log.LogCleaner)
[2020-04-22 17:57:13,270] INFO The cleaning for partition test-part-1-rep-5-0 is aborted and paused (kafka.log.LogCleaner)
[2020-04-22 17:57:13,270] INFO The cleaning for partition test-part-1-rep-5-0 is aborted (kafka.log.LogCleaner)
[2020-04-22 17:57:13,273] INFO The cleaning for partition test-part-1-rep-5-0 is aborted and paused (kafka.log.LogCleaner)
[2020-04-22 17:57:13,273] INFO The cleaning for partition test-part-1-rep-5-0 is aborted (kafka.log.LogCleaner)
[2020-04-22 17:57:13,281] INFO The cleaning for partition test-part-1-rep-5-0 is aborted and paused (kafka.log.LogCleaner)
[2020-04-22 17:57:13,281] INFO The cleaning for partition test-part-1-rep-5-0 is aborted (kafka.log.LogCleaner)
[2020-04-22 17:59:21,248] ERROR Failed to access checkpoint file cleaner-offset-checkpoint in dir /data/sdi1500059/logs/kafka-logs-3 (kafka.log.LogCleaner)
org.apache.kafka.common.errors.KafkaStorageException: Error while reading checkpoint file /data/sdi1500059/logs/kafka-logs-3/cleaner-offset-checkpoint
Caused by: java.nio.file.NoSuchFileException: /data/sdi1500059/logs/kafka-logs-3/cleaner-offset-checkpoint
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
	at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:214)
	at java.nio.file.Files.newByteChannel(Files.java:361)
	at java.nio.file.Files.newByteChannel(Files.java:407)
	at java.nio.file.spi.FileSystemProvider.newInputStream(FileSystemProvider.java:384)
	at java.nio.file.Files.newInputStream(Files.java:152)
	at java.nio.file.Files.newBufferedReader(Files.java:2784)
	at java.nio.file.Files.newBufferedReader(Files.java:2816)
	at kafka.server.checkpoints.CheckpointFile.liftedTree2$1(CheckpointFile.scala:87)
	at kafka.server.checkpoints.CheckpointFile.read(CheckpointFile.scala:86)
	at kafka.server.checkpoints.OffsetCheckpointFile.read(OffsetCheckpointFile.scala:61)
	at kafka.log.LogCleanerManager.$anonfun$allCleanerCheckpoints$2(LogCleanerManager.scala:135)
	at scala.collection.TraversableLike.$anonfun$flatMap$1(TraversableLike.scala:244)
	at scala.collection.Iterator.foreach(Iterator.scala:941)
	at scala.collection.Iterator.foreach$(Iterator.scala:941)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1429)
	at scala.collection.MapLike$DefaultValuesIterable.foreach(MapLike.scala:213)
	at scala.collection.TraversableLike.flatMap(TraversableLike.scala:244)
	at scala.collection.TraversableLike.flatMap$(TraversableLike.scala:241)
	at scala.collection.AbstractTraversable.flatMap(Traversable.scala:108)
	at kafka.log.LogCleanerManager.$anonfun$allCleanerCheckpoints$1(LogCleanerManager.scala:133)
	at kafka.utils.CoreUtils$.inLock(CoreUtils.scala:253)
	at kafka.log.LogCleanerManager.allCleanerCheckpoints(LogCleanerManager.scala:141)
	at kafka.log.LogCleanerManager.$anonfun$grabFilthiestCompactedLog$1(LogCleanerManager.scala:172)
	at kafka.utils.CoreUtils$.inLock(CoreUtils.scala:253)
	at kafka.log.LogCleanerManager.grabFilthiestCompactedLog(LogCleanerManager.scala:169)
	at kafka.log.LogCleaner$CleanerThread.cleanFilthiestLog(LogCleaner.scala:321)
	at kafka.log.LogCleaner$CleanerThread.doWork(LogCleaner.scala:307)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2020-04-22 17:59:21,257] ERROR Failed to access checkpoint file cleaner-offset-checkpoint in dir /data/sdi1500059/logs/kafka-logs-0 (kafka.log.LogCleaner)
org.apache.kafka.common.errors.KafkaStorageException: Error while reading checkpoint file /data/sdi1500059/logs/kafka-logs-0/cleaner-offset-checkpoint
Caused by: java.nio.file.NoSuchFileException: /data/sdi1500059/logs/kafka-logs-0/cleaner-offset-checkpoint
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
	at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:214)
	at java.nio.file.Files.newByteChannel(Files.java:361)
	at java.nio.file.Files.newByteChannel(Files.java:407)
	at java.nio.file.spi.FileSystemProvider.newInputStream(FileSystemProvider.java:384)
	at java.nio.file.Files.newInputStream(Files.java:152)
	at java.nio.file.Files.newBufferedReader(Files.java:2784)
	at java.nio.file.Files.newBufferedReader(Files.java:2816)
	at kafka.server.checkpoints.CheckpointFile.liftedTree2$1(CheckpointFile.scala:87)
	at kafka.server.checkpoints.CheckpointFile.read(CheckpointFile.scala:86)
	at kafka.server.checkpoints.OffsetCheckpointFile.read(OffsetCheckpointFile.scala:61)
	at kafka.log.LogCleanerManager.$anonfun$allCleanerCheckpoints$2(LogCleanerManager.scala:135)
	at scala.collection.TraversableLike.$anonfun$flatMap$1(TraversableLike.scala:244)
	at scala.collection.Iterator.foreach(Iterator.scala:941)
	at scala.collection.Iterator.foreach$(Iterator.scala:941)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1429)
	at scala.collection.MapLike$DefaultValuesIterable.foreach(MapLike.scala:213)
	at scala.collection.TraversableLike.flatMap(TraversableLike.scala:244)
	at scala.collection.TraversableLike.flatMap$(TraversableLike.scala:241)
	at scala.collection.AbstractTraversable.flatMap(Traversable.scala:108)
	at kafka.log.LogCleanerManager.$anonfun$allCleanerCheckpoints$1(LogCleanerManager.scala:133)
	at kafka.utils.CoreUtils$.inLock(CoreUtils.scala:253)
	at kafka.log.LogCleanerManager.allCleanerCheckpoints(LogCleanerManager.scala:141)
	at kafka.log.LogCleanerManager.$anonfun$grabFilthiestCompactedLog$1(LogCleanerManager.scala:172)
	at kafka.utils.CoreUtils$.inLock(CoreUtils.scala:253)
	at kafka.log.LogCleanerManager.grabFilthiestCompactedLog(LogCleanerManager.scala:169)
	at kafka.log.LogCleaner$CleanerThread.cleanFilthiestLog(LogCleaner.scala:321)
	at kafka.log.LogCleaner$CleanerThread.doWork(LogCleaner.scala:307)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2020-04-22 17:59:21,258] ERROR Failed to access checkpoint file cleaner-offset-checkpoint in dir /data/sdi1500059/logs/kafka-logs-1 (kafka.log.LogCleaner)
org.apache.kafka.common.errors.KafkaStorageException: Error while reading checkpoint file /data/sdi1500059/logs/kafka-logs-1/cleaner-offset-checkpoint
Caused by: java.nio.file.NoSuchFileException: /data/sdi1500059/logs/kafka-logs-1/cleaner-offset-checkpoint
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
	at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:214)
	at java.nio.file.Files.newByteChannel(Files.java:361)
	at java.nio.file.Files.newByteChannel(Files.java:407)
	at java.nio.file.spi.FileSystemProvider.newInputStream(FileSystemProvider.java:384)
	at java.nio.file.Files.newInputStream(Files.java:152)
	at java.nio.file.Files.newBufferedReader(Files.java:2784)
	at java.nio.file.Files.newBufferedReader(Files.java:2816)
	at kafka.server.checkpoints.CheckpointFile.liftedTree2$1(CheckpointFile.scala:87)
	at kafka.server.checkpoints.CheckpointFile.read(CheckpointFile.scala:86)
	at kafka.server.checkpoints.OffsetCheckpointFile.read(OffsetCheckpointFile.scala:61)
	at kafka.log.LogCleanerManager.$anonfun$allCleanerCheckpoints$2(LogCleanerManager.scala:135)
	at scala.collection.TraversableLike.$anonfun$flatMap$1(TraversableLike.scala:244)
	at scala.collection.Iterator.foreach(Iterator.scala:941)
	at scala.collection.Iterator.foreach$(Iterator.scala:941)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1429)
	at scala.collection.MapLike$DefaultValuesIterable.foreach(MapLike.scala:213)
	at scala.collection.TraversableLike.flatMap(TraversableLike.scala:244)
	at scala.collection.TraversableLike.flatMap$(TraversableLike.scala:241)
	at scala.collection.AbstractTraversable.flatMap(Traversable.scala:108)
	at kafka.log.LogCleanerManager.$anonfun$allCleanerCheckpoints$1(LogCleanerManager.scala:133)
	at kafka.utils.CoreUtils$.inLock(CoreUtils.scala:253)
	at kafka.log.LogCleanerManager.allCleanerCheckpoints(LogCleanerManager.scala:141)
	at kafka.log.LogCleanerManager.$anonfun$grabFilthiestCompactedLog$1(LogCleanerManager.scala:172)
	at kafka.utils.CoreUtils$.inLock(CoreUtils.scala:253)
	at kafka.log.LogCleanerManager.grabFilthiestCompactedLog(LogCleanerManager.scala:169)
	at kafka.log.LogCleaner$CleanerThread.cleanFilthiestLog(LogCleaner.scala:321)
	at kafka.log.LogCleaner$CleanerThread.doWork(LogCleaner.scala:307)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2020-04-22 17:59:21,291] ERROR Failed to access checkpoint file cleaner-offset-checkpoint in dir /data/sdi1500059/logs/kafka-logs-2 (kafka.log.LogCleaner)
org.apache.kafka.common.errors.KafkaStorageException: Error while reading checkpoint file /data/sdi1500059/logs/kafka-logs-2/cleaner-offset-checkpoint
Caused by: java.nio.file.NoSuchFileException: /data/sdi1500059/logs/kafka-logs-2/cleaner-offset-checkpoint
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
	at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:214)
	at java.nio.file.Files.newByteChannel(Files.java:361)
	at java.nio.file.Files.newByteChannel(Files.java:407)
	at java.nio.file.spi.FileSystemProvider.newInputStream(FileSystemProvider.java:384)
	at java.nio.file.Files.newInputStream(Files.java:152)
	at java.nio.file.Files.newBufferedReader(Files.java:2784)
	at java.nio.file.Files.newBufferedReader(Files.java:2816)
	at kafka.server.checkpoints.CheckpointFile.liftedTree2$1(CheckpointFile.scala:87)
	at kafka.server.checkpoints.CheckpointFile.read(CheckpointFile.scala:86)
	at kafka.server.checkpoints.OffsetCheckpointFile.read(OffsetCheckpointFile.scala:61)
	at kafka.log.LogCleanerManager.$anonfun$allCleanerCheckpoints$2(LogCleanerManager.scala:135)
	at scala.collection.TraversableLike.$anonfun$flatMap$1(TraversableLike.scala:244)
	at scala.collection.Iterator.foreach(Iterator.scala:941)
	at scala.collection.Iterator.foreach$(Iterator.scala:941)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1429)
	at scala.collection.MapLike$DefaultValuesIterable.foreach(MapLike.scala:213)
	at scala.collection.TraversableLike.flatMap(TraversableLike.scala:244)
	at scala.collection.TraversableLike.flatMap$(TraversableLike.scala:241)
	at scala.collection.AbstractTraversable.flatMap(Traversable.scala:108)
	at kafka.log.LogCleanerManager.$anonfun$allCleanerCheckpoints$1(LogCleanerManager.scala:133)
	at kafka.utils.CoreUtils$.inLock(CoreUtils.scala:253)
	at kafka.log.LogCleanerManager.allCleanerCheckpoints(LogCleanerManager.scala:141)
	at kafka.log.LogCleanerManager.$anonfun$grabFilthiestCompactedLog$1(LogCleanerManager.scala:172)
	at kafka.utils.CoreUtils$.inLock(CoreUtils.scala:253)
	at kafka.log.LogCleanerManager.grabFilthiestCompactedLog(LogCleanerManager.scala:169)
	at kafka.log.LogCleaner$CleanerThread.cleanFilthiestLog(LogCleaner.scala:321)
	at kafka.log.LogCleaner$CleanerThread.doWork(LogCleaner.scala:307)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
[2020-04-22 17:59:21,980] ERROR Failed to access checkpoint file cleaner-offset-checkpoint in dir /data/sdi1500059/logs/kafka-logs-4 (kafka.log.LogCleaner)
org.apache.kafka.common.errors.KafkaStorageException: Error while reading checkpoint file /data/sdi1500059/logs/kafka-logs-4/cleaner-offset-checkpoint
Caused by: java.nio.file.NoSuchFileException: /data/sdi1500059/logs/kafka-logs-4/cleaner-offset-checkpoint
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
	at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:214)
	at java.nio.file.Files.newByteChannel(Files.java:361)
	at java.nio.file.Files.newByteChannel(Files.java:407)
	at java.nio.file.spi.FileSystemProvider.newInputStream(FileSystemProvider.java:384)
	at java.nio.file.Files.newInputStream(Files.java:152)
	at java.nio.file.Files.newBufferedReader(Files.java:2784)
	at java.nio.file.Files.newBufferedReader(Files.java:2816)
	at kafka.server.checkpoints.CheckpointFile.liftedTree2$1(CheckpointFile.scala:87)
	at kafka.server.checkpoints.CheckpointFile.read(CheckpointFile.scala:86)
	at kafka.server.checkpoints.OffsetCheckpointFile.read(OffsetCheckpointFile.scala:61)
	at kafka.log.LogCleanerManager.$anonfun$allCleanerCheckpoints$2(LogCleanerManager.scala:135)
	at scala.collection.TraversableLike.$anonfun$flatMap$1(TraversableLike.scala:244)
	at scala.collection.Iterator.foreach(Iterator.scala:941)
	at scala.collection.Iterator.foreach$(Iterator.scala:941)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1429)
	at scala.collection.MapLike$DefaultValuesIterable.foreach(MapLike.scala:213)
	at scala.collection.TraversableLike.flatMap(TraversableLike.scala:244)
	at scala.collection.TraversableLike.flatMap$(TraversableLike.scala:241)
	at scala.collection.AbstractTraversable.flatMap(Traversable.scala:108)
	at kafka.log.LogCleanerManager.$anonfun$allCleanerCheckpoints$1(LogCleanerManager.scala:133)
	at kafka.utils.CoreUtils$.inLock(CoreUtils.scala:253)
	at kafka.log.LogCleanerManager.allCleanerCheckpoints(LogCleanerManager.scala:141)
	at kafka.log.LogCleanerManager.$anonfun$grabFilthiestCompactedLog$1(LogCleanerManager.scala:172)
	at kafka.utils.CoreUtils$.inLock(CoreUtils.scala:253)
	at kafka.log.LogCleanerManager.grabFilthiestCompactedLog(LogCleanerManager.scala:169)
	at kafka.log.LogCleaner$CleanerThread.cleanFilthiestLog(LogCleaner.scala:321)
	at kafka.log.LogCleaner$CleanerThread.doWork(LogCleaner.scala:307)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:89)
